test/integration/isolated/suite_test.go

```diff
@@ -243,7 +243,7 @@ func featureSetup(opts ...featureSetupOpt) func(ctx context.Context, t *testing.
 			}
 
 			return ok
-		}, time.Minute, 100*time.Millisecond, "failed waiting for kong addon to become ready") {
+		}, time.Minute*3, 100*time.Millisecond, "failed waiting for kong addon to become ready") {
```

Comment:
Why should we update the timeout to 3 minutes here? Is there some cases where Kong addon becomes ready between 1 minute and 3 minutes?

Reply:
In my local setup `1m` was on the edge and I was getting timeouts sometimes. It's probably because of my recent issues with network bandwidth. If you think it's too long for this timeout, I'm okay with changing this back.